packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -79,11 +79,31 @@ function registerCommands(): vscode.Disposable {
       return launchFromLogFile(lastOpenedLog);
     }
   );
-  return vscode.Disposable.from(
-    promptForLogCmd,
-    launchFromLogFileCmd,
-    launchFromLastLogFileCmd
-  );
+
+  if (checkpointsEnabled) {
+    const sfdxCreateCheckpointsCmd = vscode.commands.registerCommand(
+      'sfdx.create.checkpoints',
+      sfdxCreateCheckpoints
+    );
+    const sfdxToggleCheckpointCmd = vscode.commands.registerCommand(
+      'sfdx.toggle.checkpoint',
+      sfdxToggleCheckpoint
+    );
```

Comment:
I think you could remove this check to register the command by attaching a when clause to the context being set for checkpoints in package.json. We do this as well to verify sfdx projects are opened and if the replay extension is installed

Reply:
@Sweetman This won't be an issue when checkpoints are fully enabled. Right now I'm just leaving this the way it is.